src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -129,7 +129,7 @@ <h3 class="h3" id="see">{% trans "Zie ook" %}</h3>
 
             {% render_column span=9 %}
                 {% if object.contacts.exists %}
-                    {% render_notification type='success' icon=False %}
```

Comment:
Looks like this was the only (remaining) use of this template-tag so we could remove it.